content/using_bookwyrm/main-menu.md

```diff
@@ -0,0 +1,123 @@
+
+`user@example.com` or `arthur@ramblingreaders.org`
+
+### Home Timeline
+
+Your Home Timeline is a chronological feed of all the statuses posted by people you follow.
+This includes their reading status updates (want to read, started reading, finished, etc), reviews, ratings, comments, and quotes that they've shared.
+Read more about [types of statuses here](/posting-statuses.html).
+
+Your Home Timeline also includes any statuses that people you follow have **boosted**.
+This is similar to reposting or reblogging on other platforms, and means that status has been shared into your Timeline.
+Boosting is the best way to share good reviews with your followers, and helps them find interesting people to follow.
+This is especially useful since there is no algorithm deciding what to show you,
+only the real humans that you've decided to follow.
+
+### Books Timeline
+
+Your Books Timeline is a chronological feed that collects statuses posted about books that you have in [your shelves](/shelves.html).
+
+It might not display *all* statuses, if your BookWyrm website doesn't know about some
+
+### Interacting with statuses
+
+You can reply to any status in your timeline to start a discussion, and boost or like the status.
+Under 'More Options' there is a link to send a direct message to the poster, report the status, or block them.
+
+### Filtering Timelines
+
+Both timelines have a Filters menu which lets you filter what types of statuses you'd like to see in it.
+You can include or exclude Reviews, Comments, Quotations, or Everything Else (including reading status updates). 
+Then click the 'Apply filters' button to reload the timeline with the new filters.
+
+A little badge called 'Filters are applied' will be shown after filters are changed, 
+but this disappears after another page reload even though filters are still being applied.
+
+## Navigation to other pages
+
+There are many links within the main menu to navigate to other pages on BookWyrm.
+
+### Top menu bar
+
+In the top bar is the search box to go to the search page, and links to [Lists](/lists.html), 
+Discover (recent statuses from the local instance), and [Your Books](/shelves.html).
+
+On the right hand side is your profile icon and name. 
+When clicked, it has links to your Profile, the Directory of other users, your Direct Messages, Settings, and the Log Out button.
+
+On the far right is the Notifications button. 
+It will display a number if you have unread notifications.
+
+On mobile, some of these options will be collapsed into a 3 dots option which will show them.
+
+### Footer
+
+The website footer also includes links related to the administration of the site, link to this documentation and to re-start the Guided Tour. 
+
+It might also include a link to financially support the admins who provide you this service for free,
+even though it costs them many hours of management and up to several hundred dollars a year to run.
```

Comment:
No need for a new line.

Not sure about this sentence, actually - There's nothing stopping someone running a BookWyrm instance as a private club you have to pay to access, or some other model.